go/interpreter/lfvm/interpreter_test.go

```diff
@@ -417,45 +416,69 @@ func TestRunGenerateResult(t *testing.T) {
 	baseGas := tosca.Gas(2)
 	baseRefund := tosca.Gas(3)
 
-	getCtxt := func() context {
-		ctxt := getEmptyContext()
-		ctxt.gas = baseGas
-		ctxt.refund = baseRefund
-		ctxt.memory = NewMemory()
-		ctxt.memory.store = baseOutput
-		ctxt.resultSize = uint256.Int{uint64(len(baseOutput))}
-		return ctxt
-	}
-
 	tests := map[string]struct {
-		setup          func(*context)
+		status         status
+		returnData     []byte
 		expectedErr    error
 		expectedResult tosca.Result
 	}{
-		"max init code": {func(ctx *context) { ctx.status = statusError }, nil,
-			tosca.Result{Success: false}},
-		"error": {func(ctx *context) { ctx.status = statusError }, nil, tosca.Result{Success: false}},
-		"returned": {func(ctx *context) { ctx.status = statusReturned }, nil, tosca.Result{Success: true,
-			Output: baseOutput, GasLeft: baseGas, GasRefund: baseRefund}},
-		"reverted": {func(ctx *context) { ctx.status = statusReverted }, nil,
-			tosca.Result{Success: false, Output: baseOutput, GasLeft: baseGas, GasRefund: 0}},
-		"stopped": {func(ctx *context) { ctx.status = statusStopped }, nil,
-			tosca.Result{Success: true, Output: nil, GasLeft: baseGas, GasRefund: baseRefund}},
-		"suicide": {func(ctx *context) { ctx.status = statusSelfDestructed }, nil,
-			tosca.Result{Success: true, Output: nil, GasLeft: baseGas, GasRefund: baseRefund}},
-		"unknown status": {func(ctx *context) { ctx.status = statusError + 1 },
-			fmt.Errorf("unexpected error in interpreter, unknown status: %v", statusError+1), tosca.Result{}},
-		"getOutput fail": {func(ctx *context) {
-			ctx.status = statusReturned
-			ctx.resultSize = uint256.Int{1, 1}
-		}, nil, tosca.Result{Success: false}},
+		"max init code": {
+			status:         statusError,
+			expectedResult: tosca.Result{Success: false}},
+		"error": {
+			status:         statusError,
+			expectedResult: tosca.Result{Success: false}},
```

Comment:
These two test cases are identical ...

Reply:
will delete max init code, since it is covered in error case as well.